beyond_django/settings.py

```diff
@@ -68,6 +68,25 @@
     },
 ]
 
+LOGGING = {
```

Comment:
please don't touch this - the defaults should be enough for now

Reply:
Ack